File: app/src/main/java/org/bluechat/blueninemenmoris/AchievementsActivity.java
package org.bluechat.blueninemenmoris;

import android.app.Activity;
import android.content.SharedPreferences;
import android.graphics.Typeface;
import android.media.AudioManager;
import android.media.SoundPool;
import android.os.Bundle;
import android.preference.PreferenceManager;
import android.view.Window;
import android.widget.TextView;


/**
 * Created by Samsaini on 12/05/2016.
 */
public class AchievementsActivity extends Activity{
    private static final String TAG = "AchiActivity" ;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // Setup the window
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        requestWindowFeature(Window.FEATURE_INDETERMINATE_PROGRESS);
        setContentView(R.layout.achievements);
        Typeface typeface = Typeface.createFromAsset(getAssets(),
                "CarterOne.ttf");
        Typeface typeface1 = Typeface.createFromAsset(getAssets(),
                "future.otf");
        ((TextView) findViewById(R.id.a1)).setTypeface(typeface1);
        ((TextView) findViewById(R.id.a2)).setTypeface(typeface);
        TextView win = ((TextView) findViewById(R.id.a3));
        win.setTypeface(typeface);
        win.setText(Settings.gamewin + " Wins");
        TextView lost = ((TextView) findViewById(R.id.a4));
        lost.setTypeface(typeface);
        lost.setText(Settings.gamelost + " Lost");

    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: app/src/main/java/org/bluechat/blueninemenmoris/AchievementsActivity.kt
package org.bluechat.blueninemenmoris

import android.graphics.Typeface
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import android.view.Window
import android.widget.TextView


/**
 * Created by Samsaini on 12/05/2016.
 */
class AchievementsActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Setup the window
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        requestWindowFeature(Window.FEATURE_INDETERMINATE_PROGRESS)
        setContentView(R.layout.achievements)
        val typeface = Typeface.createFromAsset(assets,
                "CarterOne.ttf")
        val typeface1 = Typeface.createFromAsset(assets,
                "future.otf")
        (findViewById<View>(R.id.a1) as TextView).setTypeface(typeface1)
        (findViewById<View>(R.id.a2) as TextView).setTypeface(typeface)
        val win = findViewById<View>(R.id.a3) as TextView
        win.setTypeface(typeface)
        win.text = Settings.gamewin.toString() + " Wins"
        val lost = findViewById<View>(R.id.a4) as TextView
        lost.setTypeface(typeface)
        lost.text = Settings.gamelost.toString() + " Lost"

    }

    override fun onDestroy() {
        super.onDestroy()
    }

    companion object {
        private val TAG = "AchiActivity"
    }
}
